### ISSUE TITLE
Support for multiple "only"

### ISSUE THREAD

**Issue by tpburch** [hapijs/lab#922]
There is some odd behavior around `only` flags.

I have found in recent versions of lab (>= 18 - maybe earlier), that I can put `only` on multiple tests and they will all run. My assumption was that this was an undocumented new feature. However, I also found that this does not work when the elements marked `only` share a common experiment that is > 2 levels higher. This is due to the implementation of `internals.skipAllButOnly` in runner.js. 

So, I was planning to submit a PR that allows multiple `only` flags regardless of depth. Then I came across the test:
```javascript
it('reports an error if there is more than one "only", even accross multiple scripts', async () => {
```
According to this test, multiple `only` flags should always throw an error. This test passes, but it passes because the assertions are being skipped:
```javascript
        try {
            await Lab.execute([script1, script2], {}, null);
        }
        catch (ex) {
            expect(ex.message).to.contain([
                'Multiple tests are marked as "only":',
                'Test: test a',
                'Experiment: test2'
            ]);
        }
```
Based on the current implementation, this test passes because the `execute` doesn't reject, so nothing gets caught. If I add a `Code.fail()` or `throw new Error()` at the end of the `try` block, the test then fails.

So, what I'd _like_ is for lab to support multiple `only` flags - and run every test/experiment that is flagged with `only`. If this seems reasonable, I can make a small PR to implement this behavior.

If not, then there is likely a bug to be filed/fixed, since multiple `only` flags are currently respected in _some_ cases, but not others.

Thoughts?

**Comment by geek** [comment]
@tpburch thank you for raising this. Your expectation is the same as mine, every `only` will execute. This particular test case looks like a mistake and should be removed or fixed.

**Comment by tpburch** [comment]
@geek that being the case, multiple `only`s doesn't _completely_ work right now. I could work up a PR that would include removing the erroneous test, if that seems reasonable?

**Comment by geek** [comment]
@tpburch that will be fantastic if you can create a PR to fix the current behavior.

**Comment by tpburch** [comment]
https://github.com/hapijs/lab/pull/923

**Comment by geek** [comment]
Fixed by #923
